### DISCUSSION TITLE
“.lotus/datastore/splitstore” Unlimited folder growth。

### DISCUSSION THREAD

**Discussion by ChaiWenyu1983** [filecoin-project/lotus#9905] [Storage Provider] [answered]
“.lotus/datastore/splitstore” Unlimited folder growth，the disk gets saturated and causes failures. It grows at a visible rate every day, up to 700G. Two folders "hot. bagger. 1..." will be generated. How can I prevent this?

**Comment by TippyFlitsUK** [comment, marked as answer]
Hi @ChaiWenyu1983 👋 
Can you please download a new lightweight snapshot and set `HotStoreFullGCFrequency` to a lower number? Please make sure you delete the chain folder and clean the splitstore folder before importing the snapshot - `lotus-shed splitstore clean`

**Comment by Ferrydh** [comment]
这是来自QQ邮箱的假期自动回复邮件。
 
您好，我最近正在休假中，无法亲自回复您的邮件。我将在假期结束后，尽快给您回复。
